lib/job/compute-next-run-at.ts

```diff
@@ -36,14 +36,16 @@ export const computeNextRunAt = function(this: Job) {
     debug('[%s:%s] computing next run via interval [%s]', this.attrs.name, this.attrs._id, interval);
     let lastRun = this.attrs.lastRunAt || new Date();
     lastRun = dateForTimezone(lastRun);
+    var cronOptions: any = { currentDate: lastRun.toDate() };
```

Comment:
Nit, could you switch this `var` to `const` to keep code consistent? Thanks!

Reply:
Good catch, I will fix the PR.